services/imageproxy/imageproxy_test.go

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package imageproxy
```

Comment:
Are there meant to be tests here, or is there nothing it makes sense to test in isolation here?

Reply:
Oops, I didn't meant to check this in. I'll see if there's anything that makes sense to test here. Maybe something around changing the backends

Reply:
I don't have time to add any more tests right now (since I spent time adding a bunch of other unnecessary tests :p), so I just removed this file for the time being. If we revisit it, we can add tests for the backend changing logic